Doc/howto/descriptor.rst

```diff
@@ -42,7 +42,7 @@ add new capabilities one by one.
 Simple example: A descriptor that returns a constant
 ----------------------------------------------------
 
-The :class:`Ten` class is a descriptor whose :meth:`__get__` method always
+The ``Ten`` class is a descriptor whose :meth:`__get__` method always
```

Comment:
If we keep the `class` role but add an exclamation mark then we keep the formatting of a class but don't try to resolve the reference, and get no warning:

```suggestion
The :class:`!Ten` class is a descriptor whose :meth:`!__get__` method always
```

Re: https://devguide.python.org/documentation/markup/#roles

Same applies to the others in this PR.

And we can also do it for the example `__get__` method here.

Reply:
@hugovk Fixed class references by using `!` prefix. `__get__` is documented in <https://docs.python.org/3/reference/datamodel.html#object.__get__> , so I think it is better to fix `__get__` to `~object.__get__`.

Reply:
> `__get__` is documented in [docs.python.org/3/reference/datamodel.html#object.__get__](https://docs.python.org/3/reference/datamodel.html#object.__get__) , so I think it is better to fix `__get__` to `~object.__get__`.

Yes, good idea.